config/blueprint.php

```diff
@@ -170,6 +170,7 @@
         'notification' => \Blueprint\Generators\Statements\NotificationGenerator::class,
         'resource' => \Blueprint\Generators\Statements\ResourceGenerator::class,
         'view' => \Blueprint\Generators\Statements\ViewGenerator::class,
+        'inertia_page' => \Blueprint\Generators\Statements\InertiaPageGenerator::class,
```

Comment:
Is this the official name of these "views". Or could we simply call it `inertia`?

Reply:
This is the official name they use in their docs: https://inertiajs.com/pages#creating-pages

But yeah, nothing prevents us from simply saying:

```suggestion
        'inertia' => \Blueprint\Generators\Statements\InertiaGenerator::class,
```

As you prefer!

Reply:
If that's what they use, then no problem.